packages/react-ui/components/Link/__stories__/Link.stories.tsx

```diff
@@ -45,3 +46,23 @@ Disabled.story = { parameters: { creevey: { tests: linkTests } } };
 
 export const WithOnClick = () => <Link onClick={() => Toast.push('Clicked!')}>Simple Link</Link>;
 WithOnClick.story = { name: 'With onClick', parameters: { creevey: { skip: [true] } } };
+
+export const Loading: CSFStory<JSX.Element> = () => (
+  <Gapped vertical>
+    <Link loading>Simple loading </Link>
+    <div style={{ width: '300px', border: '1px solid lightgrey', padding: '5px' }}>
+      {'Some long text '}
+      <Link loading>loading link </Link>
+      and end of line
+    </div>
+    <div style={{ width: '150px', border: '1px solid lightgrey', padding: '5px' }}>
+      {'Some long text '}
+      <Link loading>loading link </Link>
+      and end of line
+    </div>
+    <Link loading icon={<OkIcon />}>
+      Loading link with icon
+    </Link>
+  </Gapped>
+);
+Simple.story = { parameters: { creevey: { tests: linkTests } } };
```

Comment:
```suggestion
Loading.story = { parameters: { creevey: { tests: linkTests } } };
```

Здесь видимо `Loading.story` имелся в виду. Не хватает скриншотов в IE. А в остальном кажется все круто.